doc/ref_model/chapters/chapter03.md

```diff
@@ -263,7 +265,20 @@ Principles that should be followed during the development and definition of the
 * Future proof: Network model is extendible to support known and emerging technology trends including SmartNICs, FPGAs and Programmable Switches, integrated for multi-clouds, and Edge related technologies.
 
 <a name="3.6"></a>
-## 3.6 Sample reference model realization
+## 3.6 Storage
```

Comment:
@petorre do some of the capabilities specified in Table 3.3 also need to be covered here. For example, data availability, performance and enhanced features?

Reply:
I was thinking about that also, but to keep it short only wrote "_various storage types and performance requirements (more in Virtual Resources 3.2.3 Storage)_" with link to text above on the same page.